### ISSUE TITLE
Remove locales that aren't supported

### ISSUE THREAD

**Issue by Delphine** [mozilla-mobile/firefox-echo-show#272]
Seems like a bunch of non-supported (and so, non-shipping) locales are still present in the Firefox Echo Show code base. Since there is no in-app locale switcher, there's not much sense in keeping them.
The list of Amazon shipping locales we were given at the time were:
zh-CN (Simplified chinese)
fr (French)
de (German)
it (Italian)
ja (Japanese)
pt-BR (Brazilian Portuguese)
es-MX (Mexican Spanish)

@athomasmoz Can you please confirm this is still the current list we need?
(also CC @Pike and @mcomella for info)
thanks!

**Comment by athomasmoz** [comment]
Hey Delphine! Yes, thanks for calling this out. Alexa only supports English, German, and Japanese right now

**Comment by Delphine** [comment]
@athomasmoz Thanks for your response. At the time, we were specifically requested to add the Mexican Spanish localization to Echo Show (adapting from the FFTV localization of Spanish from Spain).
I think that request came from @liuche at the time. Just want to be extra sure because we were definitely requested more languages when the project kicked-off :)

**Comment by liuche** [comment]
Yeah, that's true, we did ask for es-MX and had Jeff Beatty help us look over the translations in person - @devinreams can you follow up on this? I don't know if the situation has changed, or if we still need these languages on the Echo Show.

**Comment by Delphine** [comment]
Circling back about this (sorry, this is just a copy/paste from my comment on FFtv since it's the same scenario, basically)
I've discussed this with my team and we are a bit concerned because the mobile team forked all these locales/strings from an old repo that is now unmaintained. And we don't really know what's in it at this point.
The ideal solution here would be to remove all the "useless" locales - and start fresh on the locales you need (as per statement "Need to prioritize the additional locales in repo").
I think Axel has already discussed about doing this with @mcomella not long ago.

(CC @flod since so he's in the loop)

**Comment by mcomella** [comment]
wrt Delphine's comment, I followed up on the FFTV issue: https://github.com/mozilla-mobile/firefox-tv/issues/1975#issuecomment-477340663 Let's keep the discussion there for now.

**Comment by mcomella** [comment]
Assigning @devinreams to follow up on required locales, as per https://github.com/mozilla-mobile/firefox-echo-show/issues/272#issuecomment-474459941

Note: I removed the unused locales in FFTV so I know how to do it quickly here. Basically, I ran some python interpreter commands.
